cognite_toolkit/_cdf_tk/prototypes/interactive_init.py

```diff
@@ -97,33 +115,36 @@ def interactive(
                 help="Directory path to project to initialize or upgrade with templates.",
             ),
         ] = None,
-        arg_selected: Annotated[
+        package: Annotated[
```

Comment:
Makes it easier to read in the code below, you can always a different name of the parameter in the CLI by setting it in `typer.Option("--package'', ...)`.
```suggestion
        package_name: Annotated[
```